setup.py

```diff
@@ -7,8 +7,8 @@
 setup(
     name='clcache',
     description='MSVC Complier cache',
```

Comment:
There's a typo here ("Complier") and I suppose "compiler" should be written all lower-case.